### DISCUSSION TITLE
How to build UTM locally to enable increased memory?

### DISCUSSION THREAD

**Discussion by YazeedAlKhalaf** [utmapp/UTM#4425] [UTM Help (iOS)]
I tried building the app but I am facing this issue before the build starts, I get an error stating that Hypervisor is not available when building for iOS.
I couldn't figure it out and it is not mentioned in any documentation.
I tried making the entitlement that enables it anywhere false but the same issue.
I have a paid developer account so that is not the problem.

I want to build it for an iPad Pro wih 8gb ram.

Image for issue:
![Screen Shot 2022-09-16 at 9 33 06 PM](https://user-images.githubusercontent.com/41989385/190707946-11a0dad4-d435-4911-9de9-bc05f288e16c.png)

I am not so familiar with iOS development so I am not sure how to remove the hypervisor or how it is being built in the ci successfully.

UPADTE 1: tried building using the command line script and it yields the same error.

**Comment by osy** [comment]
First make sure you’re using Xcode 14 and next make sure you’ve downloaded the sysroot from the latest release. Then you can use build_utm.sh or Xcode by following the documentation for iOS building in the documentation folder.

**Reply by YazeedAlKhalaf** [reply]
where do i place the sysroot?

**Reply by YazeedAlKhalaf** [reply]
after going through the repository scripts and .gitignore, it seems i need to put it in the repository root.
can you verify what i am doing is correct?

<img width="281" alt="Screen Shot 1444-02-21 at 12 31 59 AM" src="https://user-images.githubusercontent.com/41989385/190785568-806dc013-91e1-423e-a25a-ebb4476454ec.png">

**Reply by osy** [reply]
Read the development guide.

**Reply by YazeedAlKhalaf** [reply]
thanks for the reply, i read it but some of the instructions aren't so clear.
I suggest adding:
- xcode version to use for build
- explaining what is sysroot, i had to google it

i learned a lot but it could've been easier.

i appreciate your help!
